jib-cli/README.md

```diff
@@ -56,16 +56,16 @@ Most users should download a ZIP archive (Java application). We are working on r
 
 A JRE is required to run this Jib CLI distribution.
 
-Find the [latest jib-cli 0.11.0 release](https://github.com/GoogleContainerTools/jib/releases/latest) on the [Releases page](https://github.com/GoogleContainerTools/jib/releases) and download `jib-jre-<version>.zip`.
+Find the [latest jib-cli 0.12.0 release](https://github.com/GoogleContainerTools/jib/releases/latest) on the [Releases page](https://github.com/GoogleContainerTools/jib/releases) and download `jib-jre-<version>.zip`.
 
 Unzip the zip file. The zip file contains the `jib` (`jib.bat` for Windows) script at `jib/bin/`. Optionally, add the binary directory to your `$PATH` so that you can call `jib` from anywhere.
 
 We generate [SLSA3 signatures](https://slsa.dev/) using the OpenSSF's [slsa-framework/slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator) during the release process. To verify a release binary:
 1. Install the verification tool from [slsa-framework/slsa-verifier#installation](https://github.com/slsa-framework/slsa-verifier#installation).
-2. Download the signature file `attestation.intoto.jsonl` from the [GitHub releases page](https://github.com/GoogleContainerTools/jib/releases/latest).
+2. Download the signature file `jib-jre-<version>.zip.intoto.jsonl` from the [GitHub releases page](https://github.com/GoogleContainerTools/jib/releases/latest).
```

Comment:
I could also be missing some background knowledge for this but it looks like the signature file in the most recent release is still `jib-jre-<version>.zip`?

Reply:
Yup - it used to be `attestation.intoto.jsonl` (in the 0.11.0 release) but the generated filename is now changed to jib-jre-0.12.0.zip.intoto.jsonl. I checked and saw that the [release notes for v1.2.1 of slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator/releases/tag/v1.2.1) mentions this change under `Provenance file names`.

Reply:
Oh I see! Thank you.